hack/istio_peer_auth.yaml

```diff
@@ -1,8 +1,43 @@
 # For testing Istio.  Apply this to the namespace to set the mTLS policy.
+# https://istio.io/latest/docs/tasks/security/authentication/authn-policy/#enable-mutual-tls-per-workload
+#
```

Comment:
I feel I have seen these changes before. Didn't you merge these changes in the istio PR?